### PR TITLE
Updated Automatically admit people section in meeting-policies-in-teams.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7276.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-in-teams.md [hunk 1]
```diff
@@ -463,10 +463,11 @@ This is a per-organizer policy. This setting controls whether people join a meet
...
-|**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations and anonymous users must wait in the lobby.           |
+|**People in my organization, trusted organizations, and guests**     |Authenticated users within the organization, including guest users and the users from trusted organizations, join the meeting directly without waiting in the lobby. Anonymous users wait in the lobby.   |
 |**People in my organization**  |Authenticated users from within the organization, excluding guest users, join the meeting directly without waiting in the lobby. Guests and users from trusted organizations and anonymous users wait in the lobby.|
+|**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "Only me".            |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:469]:
```suggestion
|**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "Only me".            |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
Closing this one as it's very old and these improvements were handled and being handling here:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7339
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7674

**Pull request closed without merging**